### PR TITLE
[Part 1] Layout structuring and slicing

### PR DESCRIPTION
**Pull request by araraltawarah98**

### CODE REVIEW

File: task_1/index.html [hunk 1]
```diff
@@ -1,13 +1,48 @@
...
+      <div class="coffee-item"></div>
+      <div class="coffee-item"></div>
+      <div class="coffee-item"></div>
+      <div class="coffee-item"></div>
+      <div class="coffee-item"></div>
+    </div>
+  </div>
 </body>
+
 </html>
```
Comment (abood454) [inline task_1/index.html:48]: add space

File: task_1/index.html [hunk 2]
```diff
@@ -1,13 +1,48 @@
...
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Task 1</title>
   <link rel="stylesheet" href="./style.css">
-  <title>Document</title>
 </head>
+
 <body>
-  <script src="./index.js"></script>
```
Comment (abood454) [inline task_1/index.html:11]: the same link!

File: task_1/style.css [hunk 3]
```diff
@@ -1,3 +1,50 @@
-body {
```
Comment (abood454) [inline task_1/style.css:1]: } is missing

**Review by abood454 — COMMENTED** [review]
good job

File: task_1/index.html [hunk 4]
```diff
@@ -1,13 +1,48 @@
...
+  <script src="./index.js">
```
Comment (laithyounes) [inline task_1/index.html:13]: let's move the script tag to the bottom

File: task_1/index.html (hunk 1 above)
Comment (laithyounes) [inline task_1/index.html:48]: yep, end of file line

**Review by laithyounes — CHANGES_REQUESTED** [review]
please update the requested changes, 
other than that LGTM

File: task_1/style.css [hunk 5]
```diff
@@ -1,64 +1,50 @@
...
-  grid-gap: 10px;
-}
-
-.grid > div {
-  background-color: darkorange;
-}
+  grid-gap: 32px;
+  grid-template-columns: repeat(auto-fill, minmax(270px, 1fr));
+  padding-top: 3rem;
+}
```
Comment (ashrafabuodeh) [inline task_1/style.css:50]: add space

**Review by Mohsen-Awad — APPROVED** [review]
Good work

**Review by laithyounes — APPROVED** [review]

**Review by abood454 — APPROVED** [review]
WOW

**Review by ashrafabuodeh — APPROVED** [review]

**Review by Mohsen-Awad — APPROVED** [review]

**Pull request merged**